src/main.c

```diff
@@ -91,7 +92,7 @@ static void after_moving(void)
  *
  * @param[in] run Whether the robot should be running.
  */
-static void movement_phase(bool do_run)
+static void configure_speed(bool do_run)
```

Comment:
Currently, this function is not only configuring the speed, but it is also moving the robot. I think that the name and description is unclear. It is hard to follow the code (comparing with previous exploration-movement phase functions)

Reply:
[Reply on a different comment]